.travis.yml

```diff
@@ -7,7 +7,7 @@ env:
 before_install:
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
-- pip install --upgrade pip  # pip-tools requires pip >= 6.1
+- pip install --upgrade pip==9.0.3  # pip-tools requires pip >= 6.1 & pin to 9.1.3 until tox-battery upgrades
```

Comment:
It looks like I originally gave you the wrong version number, you fixed it in the code, but left it unchanged in the comment and PR title.  I accept blame for this, but can we make these consistent? 😃 Otherwise looks fine.

Reply:
Haha yes, good catch :) I blame copy/paste/commiting 5 minutes before a meeting.